app/Resources/TwigBundle/views/Exception/error.html.twig

```diff
@@ -0,0 +1,54 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+{% extends '::base.html.twig' %}
+
+{% block stylesheets %}
+  <link rel="stylesheet" href="{{ asset('themes/new-theme/public/theme.css') }}" />
+{% endblock %}
+
+{% block title %}
+  {{ 'An unexpected error occurred'|trans({},'Admin.Catalog.Help' ) }}
+{% endblock %}
+
+{% block body %}
+  <div class="container">
+    <div class="row justify-content-md-center">
+      <div class="col-12">
+        <div class="card">
+          <div class="card-body text-center">
+            <h1 class="card-title">{{ status_text }}</h1>
+            <h3 class="card-subtitle mb-2 text-muted">{{ 'Status code:'|trans({}, 'Admin.Global') }}{{ status_code }}</h3>
+
+            {% if exception %}
+              <p class="lead mb-0">{{ 'See additional error information below.'|trans({}, 'Admin.Global') }}</p>
```

Comment:
Perhaps I would choose a more user-friendly wording here, like _Cheer up, see additional information below for help_. What do you think? @TristanLDD / @colinegin

Reply:
yes, definetly. :+1: i'd like to update UI as well to match PS styles.